scripts/trackinglogsetting.sh

```diff
@@ -0,0 +1,12 @@
+BACKUP_STORAGEACCOUNT_NAME=
```

Comment:
Why is this file necessary?

Reply:
"trackinglogsetting.sh" file is not required and removed.